core/components/atoms/dropdown/DropdownList.tsx

```diff
@@ -290,6 +303,14 @@ const DropdownList = (props: OptionsProps) => {
     maxHeight,
     overflowY: 'auto',
     overflowX: 'hidden',
+    minHeight: minHeight,
+  };
+
+  const loadersDropdownStyle: React.CSSProperties = {
```

Comment:
Please change the variable name `loaderDropdownStyle`

Reply:
done